src/config/server.rs

```diff
@@ -47,3 +48,13 @@ impl Server {
 pub struct Proxy {
   pub url: String,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct HttpSettings {
```

Comment:
```suggestion
pub struct Upstream {
```
On second thoughts, we should call this as upstream